app/src/main/res/layout/content_transaction_details.xml

```diff
@@ -193,6 +193,30 @@
             app:layout_constraintTop_toBottomOf="@id/caption_createdAt"
             tools:text="Sunday, 18 Dec 2021" />
 
+        <com.google.android.material.imageview.ShapeableImageView
+            android:id="@+id/app_icon_for_share"
+            app:srcCompat="@drawable/ic_logo"
+            android:visibility="invisible"
+            app:layout_constraintEnd_toStartOf="@+id/app_name_for_share"
+            app:layout_constraintBottom_toBottomOf="parent"
+            android:layout_width="50dp"
+            android:layout_height="50dp" />
+
+        <TextView
+            android:id="@+id/app_name_for_share"
+            android:padding="16dp"
```

Comment:
@brookmg Buddy shall we use `dimens_16` here

Reply:
Alright, will use references. For the above one as well.